twilio/resources/chat/v2/api_default.go

```diff
@@ -652,29 +652,29 @@ func ResourceServices() *schema.Resource {
 		UpdateContext: updateServices,
 		DeleteContext: deleteServices,
 		Schema: map[string]*schema.Schema{
-			"friendly_name":                    AsString(SchemaOptional),
-			"account_sid":                      AsString(SchemaOptional),
-			"consumption_report_interval":      AsInt(SchemaOptional),
-			"date_created":                     AsString(SchemaOptional),
-			"date_updated":                     AsString(SchemaOptional),
-			"default_channel_creator_role_sid": AsString(SchemaOptional),
-			"default_channel_role_sid":         AsString(SchemaOptional),
-			"default_service_role_sid":         AsString(SchemaOptional),
-			"limits":                           AsString(SchemaOptional),
-			"links":                            AsString(SchemaOptional),
-			"media":                            AsString(SchemaOptional),
-			"notifications":                    AsString(SchemaOptional),
-			"post_webhook_retry_count":         AsInt(SchemaOptional),
-			"post_webhook_url":                 AsString(SchemaOptional),
-			"pre_webhook_retry_count":          AsInt(SchemaOptional),
-			"pre_webhook_url":                  AsString(SchemaOptional),
-			"reachability_enabled":             AsBool(SchemaOptional),
-			"read_status_enabled":              AsBool(SchemaOptional),
-			"sid":                              AsString(SchemaOptional),
-			"typing_indicator_timeout":         AsInt(SchemaOptional),
-			"url":                              AsString(SchemaOptional),
-			"webhook_filters":                  AsList(AsString(SchemaOptional), SchemaOptional),
-			"webhook_method":                   AsString(SchemaOptional),
+			"friendly_name":                    AsString(SchemaComputedOptional),
+			"account_sid":                      AsString(SchemaComputed),
+			"consumption_report_interval":      AsInt(SchemaComputedOptional),
+			"date_created":                     AsString(SchemaComputed),
+			"date_updated":                     AsString(SchemaComputed),
+			"default_channel_creator_role_sid": AsString(SchemaComputedOptional),
+			"default_channel_role_sid":         AsString(SchemaComputedOptional),
+			"default_service_role_sid":         AsString(SchemaComputedOptional),
+			"limits":                           AsString(SchemaComputed),
```

Comment:
Something I noticed is this `limits` can actually be set when I try making a call using Postman but we are restricting this in terraform. Looking at the [api_definitions](https://code.hq.twilio.com/sradhakrishna/api-definitions/blob/master/base/chat/v2/service.json#L110), `limits.channel_members` and `limits.user_members` can be set but not other fields such as `actions_per_second`. Not sure how can we can handle this here.

Reply:
Ugh. Our APIs ...

Reply:
Partial fix here: https://github.com/twilio/terraform-provider-twilio/pull/40

Doesn't yet handle marshaling the read response into the proper dot-notation schema properties. That'll be next.